backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -156,6 +156,8 @@ def process_input_files(
             except Exception as e:
                 failed_files += 1
                 error_message = f"Error processing file '{file_path}'. {e}"
+                logger.error(error_message)
+                logger.error(f"Error {traceback.format_exc()}")
```

Comment:
@kirtimanmishrazipstack no need to separately log traceback like this. Follow what's been done below 

https://github.com/Zipstack/unstract/blob/main/worker%2Fsrc%2Funstract%2Fworker%2Fworker.py#L224-L229

Reply:
done @ritwik-g